java-sdk/connectors/README.md

```diff
@@ -32,6 +32,7 @@ This produces the zip files, e.g. `simple-connector.zip` located in the `build/l
 
 At this point, the generated zip could be uploaded directly to Fusion, but follow the steps below to run as a remote plugin.
 
+To target a specific SDK version, update version property in the gradle.properties file. 
```

Comment:
Would be good to specify the property name, `connectorsSDKVersion`, and even link to the file/line [here](https://github.com/lucidworks/connectors-sdk-resources/blob/221bf9106c4e029c12f418486dc00c4060353883/java-sdk/connectors/gradle.properties#L5).

Reply:
Thanks Matt. It looks like the gradle.properties does not need to be touched in order to build to a specific SDK version as long as you checkout the tag version.